Makefile

```diff
@@ -56,7 +56,9 @@ prefix_install: installables
 	mkdir -p "$(PREFIX)/Frameworks" "$(PREFIX)/bin"
 	rsync -a --delete "$(TEMPORARY_FOLDER)$(FRAMEWORKS_FOLDER)/$(OUTPUT_FRAMEWORK)" "$(PREFIX)/Frameworks/"
 	cp -f "$(TEMPORARY_FOLDER)$(BINARIES_FOLDER)/carthage" "$(PREFIX)/bin/"
-	install_name_tool -rpath "/Library/Frameworks" "@executable_path/../Frameworks/$(OUTPUT_FRAMEWORK)/Versions/Current/Frameworks/"  "$(PREFIX)/bin/carthage"
+	install_name_tool -delete_rpath "@executable_path/../Frameworks" "$(PREFIX)/bin/carthage" # Avoid duplication of "@executable_path/../Frameworks"
+	install_name_tool -rpath "/Library/Frameworks" "@executable_path/../Frameworks" "$(PREFIX)/bin/carthage"
```

Comment:
That makes sense, thanks for pointing it out! 👍 

So flipping the ordering in Xcode will be good?

👇 From

<img width="703" alt="2017-07-15 11 14 45" src="https://user-images.githubusercontent.com/909674/28235725-f2b5652e-694e-11e7-9b62-832bf15b0ee6.png">

👇 To

<img width="703" alt="2017-07-15 11 15 16" src="https://user-images.githubusercontent.com/909674/28235729-fdee1ef4-694e-11e7-9b2d-b0f06c5cec9a.png">